opm/simulators/linalg/FlowLinearSolverParameters.hpp

```diff
@@ -162,6 +164,7 @@ namespace Opm
         bool   use_cpr_;
         std::string system_strategy_;
         bool scale_linear_system_;
+	std::string linear_solver_configuration_;
```

Comment:
This will be my last comment about indentation as it seems to messed up quite a lot. Please try to fix this even if I will not
comment on it any more.